php-exercise5/src/model/model.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace model;
+class Accounts {
```

Comment:
Please create separate files for each class.

Reply:
Done